virtualmachine/vsphere/vm.go

```diff
@@ -737,7 +737,14 @@ func (vm *VM) GetIPs() ([]net.IP, error) {
 			ips = append(ips, ip)
 		}
 	}
-	return ips, nil
+	return ips, vmMo.Self.Value, nil
+}
+
+// GetIPs returns the IPs of this VM. Returns all the IPs known to the API for
+// the different network cards for this VM. Includes IPV4 and IPV6 addresses.
+func (vm *VM) GetIPs() ([]net.IP, error) {
```

Comment:
This method seems to be invoked from a few places. I got compilation errors after removing it.